### DISCUSSION TITLE
How to increase the ft.TextField height?

### DISCUSSION THREAD

**Discussion by Mayurji** [flet-dev/flet#1381] [Q&A] [answered]
The height parameter inside the ft.TextField() results in no change in height.

**Comment by FeodorFitsner** [comment]
What results are trying to achieve?

**Comment by Mayurji** [comment]
Hi @FeodorFitsner, I am trying to create application with two TextField, aligned side by side, which is capable of taking in paragraph. I am seeing samples of textfield here: https://flet.dev/docs/controls/textfield/ 

Sample image:

![image](https://user-images.githubusercontent.com/10477789/236696262-707dfe2b-9ec0-489e-996f-b705b7cfae7c.png)

**Comment by FeodorFitsner** [comment]
So, you want to stretch text vertically?

**Comment by Mayurji** [comment]
Yes, something like that.

**Comment by FeodorFitsner** [comment]
You can use `scale`:

```python
import flet as ft

def main(page: ft.Page):
    page.add(
        ft.Text("Hello, world!", size=40, scale=ft.Scale(scale_y=2.0)),
    )

ft.app(main)
```

**Reply by Mayurji** [reply]
@FeodorFitsner It doesn't serve the purpose required, as the required field is TextField. The above code renders the below image.
![image](https://github.com/flet-dev/flet/assets/10477789/198502b4-849b-43c3-beb3-c8dfa6465db3)

**Reply by Mayurji** [reply]
@ndonkoHenri, Sorry to add you here. Need a clarification on this discussion thread?

**Reply by ndonkoHenri** [reply, marked as answer]
Hey @Mayurji, check [this](https://github.com/ndonkoHenri/Flet-Samples/tree/master/Markdown%20Editor) markdown editor I built back then.
It somehow resembles what you are tying to achieve.

Hope it helps.

**Reply by Mayurji** [reply]
@ndonkoHenri , Thank you for sharing. It does help.
